pkg/config/engine/docker/docker.go

```diff
@@ -35,6 +35,18 @@ type Config map[string]interface{}
 
 var _ engine.Interface = (*Config)(nil)
 
+type dockerRuntime map[string]interface{}
+
+var _ engine.RuntimeConfig = (*dockerRuntime)(nil)
+
+func (d *dockerRuntime) GetBinPath() string {
+	dr := *d
```

Comment:
Since we return `nil` in the case when a runtime doesn't exist, we should include:
```
if d == nil {
    return ""
}
```
here.

Reply:
Agreed